bench-cfg.lua

```diff
@@ -15,10 +15,10 @@ end
 local wal_mode = arg[2]
 
 box.cfg {
-    slab_alloc_arena    = 1,
+    memtx_memory = 1073741824,
     pid_file            = "tarantool.pid",
     wal_mode            = wal_mode,
-    snap_dir = ".",
+    memtx_dir = ".",
     work_dir = "."
```

Comment:
AFAIU, both `snap_dir` and `work_dir` are current directory by default and we can just drop those options without behaviour change.

Introduced by @alyapunov in 6bf779c391a7d0b5dbfc00041d3e9a95c6e3177e and changed later in 329ee02fb8737f5002660dc14bb9818b40ae86e5. So we can ask him.

Reply:
```
2022-02-22 21:49:20.927 [335] snapshot/101/main I> done
2022-02-22 21:49:20.932 [335] main/101/bench-cfg.lua I> ready to accept requests
2022-02-22 21:49:20.932 [335] main/104/checkpoint_daemon I> started
2022-02-22 21:49:20.933 [335] main/104/checkpoint_daemon I> scheduled the next snapshot at Tue Feb 22 23:21:50 2022
Benchmarking...
HASH unsigned
----------------------------------
replaces  : 1306.07     rps
selects   : 3932160.00  rps
selrepl   : 1488.78     rps
updates   : 1523.17     rps
deletes   : 1440.65     rps
----------------------------------
HASH unsigned
----------------------------------
replaces  : 1453.70     rps
selects   : 7864320.00  rps
selrepl   : 1503.08     rps
updates   : 1640.54     rps
deletes   : 1517.55     rps
----------------------------------
HASH str
----------------------------------
replaces  : 1440.35     rps
selects   : 403298.46   rps
selrepl   : 1679.96     rps
updates   : 1585.43     rps
deletes   : 1499.39     rps
----------------------------------
Done

```

Reply:
Seems applied and seems working. Okay. I'll resolve this discussion.